src/lib/utils/renderTemplate.js

```diff
@@ -1,17 +1,22 @@
-import mapValues from 'lodash/mapValues';
 import curry from 'lodash/curry';
 import hogan from 'hogan.js';
 
 // We add all our template helper methods to the template as lambdas. Note
 // that lambdas in Mustache are supposed to accept a second argument of
 // `render` to get the rendered value, not the literal `{{value}}`. But
 // this is currently broken (see https://github.com/twitter/hogan.js/issues/222).
-function transformHelpersToHogan(helpers, compileOptions, data) {
-  return mapValues(helpers, method =>
-    curry(function(text) {
-      const render = value => hogan.compile(value, compileOptions).render(this);
-      return method.call(data, text, render);
-    })
+function transformHelpersToHogan(helpers = {}, compileOptions, data) {
+  return Object.keys(helpers).reduce(
+    (acc, helperKey) => ({
+      ...acc,
```

Comment:
perf-wise it's probably basically the same because of the low number of helpers, but this could be done with inner mutation too